cost-analyzer/templates/kubecost-turndown-template.yaml

```diff
@@ -0,0 +1,212 @@
+{{- if and .Values.turndown .Values.turndown.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "kubecost.turndownName" . }}
+  labels:
+    app: {{ template "kubecost.turndownName" . }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "kubecost.turndownName" . }}
+  labels:
+    app: {{ template "kubecost.turndownName" . }}
+rules:
```

Comment:
can we add a very visible comment to say that all of these permissions are optional and not enabled by default? 


i propose something like: "These permissions are disabled by default and are optionally enabled with the Kubecost turndown feature." And then link to docs...